submissions/Zakkarat/js-dom/app.js

```diff
@@ -0,0 +1,46 @@
+let images = ["https://ddragon.leagueoflegends.com/cdn/img/champion/splash/Jhin_0.jpg", "https://ddragon.leagueoflegends.com/cdn/img/champion/splash/MissFortune_0.jpg", "https://ddragon.leagueoflegends.com/cdn/img/champion/splash/Caitlyn_0.jpg", "https://ddragon.leagueoflegends.com/cdn/img/champion/splash/Kaisa_0.jpg"];
+let colorBody = ["#540a24", "#1d5452", "#138994", "#07103e"];
+let colorHeader = ["#94152d", "#3cada7", "#216b73", "#202c6f"];
+let colorDesk = ["#821d30", "#008e86", "#1c7782", "#071252"];
+let names = ["JHIN", "MISS FORTUNE", "CAITLYN", "KAI'SA"];
+
+function champAuto(a) {
+  for (let i = 0; i < champ.length / 2; i++) {
+    if (champ[i] == a || champ[i + 4] == a){
+      menu();
+      header.textContent = names[i];
+      text.textContent = texty[i];
+      head.style.backgroundColor = colorHeader[i];
+      desk.style.backgroundColor = colorDesk[i];
+      cont.style.backgroundColor = colorHeader[i];
+      document.body.style.backgroundColor = colorBody[i];;
```

Comment:
Hello :) Great job, i noticed inline style in your code, this is a bad practice. You do not need to correct it in this case. Just read about the inline style and try not to use them in the future )[Inline style is bad](http://www.lostsaloon.com/technology/why-is-inline-css-bad-is-it-really-that-bad/)

Reply:
Okay. Thank you!

Reply:
HEY, NOOOOOOO :D Fix it please. We have talk already a lot about inline style :) 
Don't use inline style changing. It is a bad approach for several reasons: 

 

1. First of all, a browser will apply such styling for each line separately/ Which means, that every such line of code will be a reason for running of calculations of a page and for drawing it,  so you can receive a performance problem. Read about reflow and repaint.  

2. This is an imperative way, you need to write declarative and describe what your code does, not how. This will make your code shorter and easier to maintain. 

3. Reuse of code. Saying, you will need to rotate some other stuff - you will add a similar line to another part of an application. Which is not right because of DRY. 

4. JS for logic, CSS - for styling.

Reply:
Oh, sorry for resolving this too quickly. Now replaced those inline styles with my old(but improved) class system.

Reply:
Very nice!